beta/kustomize/overlays/letsencrypt/patch-configmap.yaml

```
@@ -0,0 +1,3 @@
+- op: add
+  path: /data/OP_TLS_DOMAIN
+  value: "bridge.example.com"
```

Small thing, but for consistency with our other examples, let's stick with `scim.example.com`:

```suggestion
  value: "scim.example.com"
```